### PR TITLE
Add catch all route and render 404 page

### PR DESCRIPTION
**Pull request by gr-eg**
Very simple styles currently. I'm no designer! 👨‍🎨
screen shot 2018-04-11 at 12 20 57

### CODE REVIEW

**Comment by gr-eg** [comment]
👍 Will do that now

File: client/src/PageNotFound.vue [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  <div class="center-grid">
+    <div class="center-grid__content">
+      <div class="text-center">
+        <img class="logo" src="./assets/CoolestProjectsLogo.png" />
+        <div class="not-found">
+          <h1>
+            It looks like you tried to go to a page that does not exist!
+          </h1>
+          <h1>
+            <a href="/">Please go to our homepage and try again</a>
```
Comment (Wardormeur) [inline client/src/PageNotFound.vue]: router-link to avoid a full reload ? cc @DanielBrierton

Comment (DanielBrierton) [reply client/src/PageNotFound.vue]: Pretty sure Vue is clever enough to not cause a page refresh with this, but yeah, `<router-link>` would be best IMO.

File: client/src/assets/scss/main.scss [hunk 2]
```diff
@@ -175,6 +175,12 @@ textarea {
   font-size: 14px;
 }
 
+.not-found {
```
Comment (DanielBrierton) [inline client/src/assets/scss/main.scss]: Just after noticing this added in here. Given it only applies to PageNotFound.vue, it should probably just be put in there.

**Pull request merged**